platform/admin/core/admin-core-migration-commands/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -28,7 +28,9 @@
             <action class="org.codice.ddf.migration.commands.ExportCommand">
                 <argument ref="configurationMigrationService"/>
                 <argument ref="security"/>
-                <argument ref="defaultExportDirectoryPath"/>
+                <argument value="${ddf.home}"/>
+                <argument value="etc"/>
+                <argument value="exported"/>
```

Comment:
Please look at the patch I sent you privately. There appears to be a gap in the schema preventing lists/arrays from being directly added as arguments to command/action classes so it required a bean to collect those values. Since it did, I created one for you to encapsulate the Path generation logic.